terraform/modules/helm_init/main.tf

```diff
@@ -48,9 +48,45 @@ resource "null_resource" "install_tiller" {
 }
 
 resource "null_resource" "wait_for_tiller" {
+  triggers = {
+    build_number = "${timestamp()}"
+  }
   provisioner "local-exec" {
     command = "timeout 60 bash -c 'until kubectl get pods -n kube-system |grep tiller; do sleep 5; done'"
   }
   depends_on = [null_resource.install_tiller]
 }
 
+#########################
+# add HELM repositories
+#########################
+
+resource "null_resource" "reinit_helm_client" {
+  triggers = {
+    build_number = "${timestamp()}"
+  }
+  provisioner "local-exec" {
+    command = "helm init --client-only"
```

Comment:
I guess we should remove "helm init --client-only" from  Create_cluster pipeline if we moved it to terraform

Reply:
done